### PR TITLE
docs: added uniform id for dashboard docs

### PR DESCRIPTION
**Pull request by iamayushdas**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [ ] Bugfix
- [x] New feature provided
- [x] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

added uniform id to dashboard docs

**Related issues**

fix/resolve [#363_apisix-website](https://github.com/apache/apisix-website/issues/363)

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [ ] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

File: docs/en/latest/USER_GUIDE.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ title: User Guide
 
 The following are parts of the modules' snapshot.
 
-## Dashboard
+## Dashboard {#apisix-dashboard}
```
Comment (juzhiyuan) [inline docs/en/latest/USER_GUIDE.md:26]: Could you please add a screenshot on this first?

Comment (iamayushdas) [reply docs/en/latest/USER_GUIDE.md:26]: yes sure

Comment (iamayushdas) [reply docs/en/latest/USER_GUIDE.md:26]: image

image

image

Comment (iamayushdas) [reply docs/en/latest/USER_GUIDE.md:26]: see id changes

Comment (juzhiyuan) [reply docs/en/latest/USER_GUIDE.md:26]: ok, got it

**Review by juzhiyuan — APPROVED** [review]

**Comment by bzp2010** [comment]
Because of the reason from [https://github.com/apache/apisix/pull/4474](https://github.com/apache/apisix/pull/4474), I'll turn this PR off now. If you have any new plan, you can reopen this PR at any time.

**Pull request closed without merging**